### ISSUE TITLE
Linux Source Build: couldn't find "librevault" executable in application or system folder

### ISSUE THREAD

**Issue by budric** [Librevault/librevault#45]
Hi,
I built master branch 486051c, and installed it.  Cmake Installation log:
```
Install the project...
-- Install configuration: "RelWithDebInfo"
-- Installing: /usr/local/bin/librevault-daemon
-- Installing: /usr/local/bin/librevault-gui
-- Up-to-date: /usr/local/share/applications/Librevault.desktop
-- Up-to-date: /usr/local/share/icons/hicolor/scalable/apps/librevault.svg
```

However when running librevault-gui I get the error "couldn't find "librevault" executable in application or system folder" - which is correct because a binary by that name is never produced or installed.  

Not sure what it's looking for and can't run the gui.

Thanks.

**Comment by GamePad64** [comment]
"librevault" here must stand for "librevault-daemon". Fixed the message in 86a6aefcb5cc458f4d42195368fbcff2871f98e3.
To start correctly, "librevault-daemon" must be located in one of PATH directories. Please, check, that you have /usr/local/bin in PATH.

**Comment by budric** [comment]
Thanks a lot, changing PATH works.
